src/components/buttons/button-group/RuiButtonGroup.vue

```diff
@@ -38,6 +38,17 @@ const emit = defineEmits<{
 const slots = useSlots();
 const { modelValue, required, disabled, color, variant, size } = toRefs(props);
 
+// When using dynamic content with v-for the slot content can contain fragment,
+// Go through the fragment and always return RuiTab only
```

Comment:
just nitpick on the comment here `RuiTab` => `RuiButton`

Reply:
Oh the copy pasta :P thanks nice catch